osbs/http.py

```diff
@@ -43,7 +43,7 @@ def decoded_json(iterable):
         if isinstance(line, str):
             yield line
         else:
-            yield line.decode(requests.utils.guess_json_utf(line))
+            yield line.encode(requests.utils.guess_json_utf(line))
```

Comment:
Hmm, I see what you mean.

What it aims to do is examine the first 4 bytes of `line`, which is presumed to be the raw bytes of the reply, and work out the encoding that has been used. Then, it should decode it to a unicode object (just 'str' in Python 3).

Is `line` not what we think?